.github/workflows/helm-chart-ci.yaml

```diff
@@ -239,3 +239,44 @@ jobs:
 
       - name: Install and test example
         run: ${{ matrix.example }}/run-tests.sh
+
+  upgrade-test:
+    runs-on: ubuntu-22.04
```

Comment:
Do we need a stub in helm-chart-ci-ignore?

Reply:
Good question. Not sure. Those are only for blocking tests? Should we make this blocking too or wait  for a bit to see how well it works?

Reply:
Good point. Not going to make this blocking for now.